requirements_all.txt

```diff
@@ -648,6 +648,7 @@ hdate==0.9.3
 heatmiserV3==1.1.18
 
 # homeassistant.components.here_travel_time
+# homeassistant.components.here_weather
```

Comment:
If we refactor and make a travel_time entity integration, maybe there should be a `here` integration with sensor, travel_time and weather platforms?

Reply:
I am completely on board. This would also mean the user only has to provide its api_key a single time. I would however do that with a follow up PR.

Reply:
Reading up here and doing some review comments, stumbling across this comment.
I think introducing a new integration and next refactoring it away is not really user-friendly. Basically, I now know this PR might be a breaking change from the get go. I'm not comfortable reviewing or approving such approach.

Reply:
My comment was over a year ago, now I have learnt alot more about homeassistant. I currently don't see a convenient way to offer both integrations out of one codebase as they are fundamentally different.


One provides travel times between to places and another provides weather information for a place.
This would lead to complicated config flows and complicated code. The two apis which are being used are nothing alike besides the fact that they are both from HERE.

@frenck do you see a way on how to accomplish that with one integration?

Reply:
Alright! Lets continue as is in that case! 👍